backend/de.metas.business/src/main/java/de/metas/order/impl/OrderBL.java

```diff
@@ -684,14 +684,6 @@ public void setBPLocation(final org.compiere.model.I_C_Order order, final org.co
 			{
 				setBPartnerLocation(order, shipLocations.get(0));
 			}
-			else if (!locations.isEmpty())
-			{
-				// set to first
-				if (order.getC_BPartner_Location_ID() == 0)
```

Comment:
pls leave a comment that we are no longer setting to default location if that's not a ship location

Reply:
commented.